src/locales/en/fields.yml

```diff
@@ -3,21 +3,31 @@ drivingLicenceNumber:
   hint: "This is the long number in section 5 on your licence for example HARRI559146MJ931 "
   validation:
     default: "Enter the number as it appears on your driving licence"
-    exactlength: "Your driving licence number should be 8-16 digits long"
-    numeric: "Your driving licence number should not include letters or symbols"
+    exactLength: "Your licence number should be 16 characters long"
+    numeric: "Your licence number should not include any symbols or spaces"
+    limit: "Driving licence numbers do not start with 16 – enter the number as it appears on your driving licence"
+dvaLicenceNumber:
+  label: Driving Licence number
```

Comment:
Only a small thing, can we make the L in licence a small l to be inline with the prototype

Reply:
Pulled your changes.